lib/auth/authentication_backend.rb

```diff
@@ -181,6 +182,21 @@ def get_user(user_id:)
     raise AuthenticationBackendException.new(e.message)
   end
 
+  def update_user_email(user_id:, email:)
+    client.admin_update_user_attributes(
+      user_pool_id: user_pool_id,
+      username: user_id,
+      user_attributes: [
+        { name: "email", value: email },
+        { name: 'email_verified', value: 'True' },
+      ],
+    )
+  rescue Aws::CognitoIdentityProvider::Errors::AliasExistsException => e
+    raise AliasExistsException.new(e)
+  rescue StandardError => e
```

Comment:
would prefer if we were rescuing the AWS ServiceError exception rather than a generic StandardError